mountainduck/interface.md

```diff
@@ -207,11 +207,13 @@ A context menu in *Finder* on macOS and *File Explorer* on Windows allows variou
 ### Sync Options
 
 *Keep Offline on Local Disk* or *Delete on Local Disk* to manage caching status.
-Refer to [Sync Options](sync/index.md#keep-offline)
+Refer to [Sync Options](sync/index.md#keep-offline).
 
 ### Reload
 
-Changes from the server are not immediately visible. The directory listing in the file browser may become outdated when another application adds, removes, or modifies files on the server. Choose *Reload* from the *Finder Extension Menu* on macOS to refresh the directory listing.
+Changes from the server are not immediately visible. The directory listing in the file browser may become outdated when another application adds, removes, or modifies files on the server. 
+
+Choose *Reload* from the context menu in _Finder_ or _Windows Explorer_ to refresh the directory listing. On Windows, you can also use the _FN → F5_ key in _Windows Explorer_ to refresh the file listing.
```

Comment:
```suggestion
Choose *Reload* from the context menu in _Finder_ or _Windows Explorer_ to refresh the directory listing. On Windows, you can also use the _F5_ key in _Windows Explorer_ to refresh the file listing.
```

FN-F5 is a quirk for several keyboard manufacturers to default to their "media keys"/display brightness/keys. This can be overriden by a "Fn-Lock"-key (whatever that is on any given keyboard).